module-services/service-desktop/WorkerDesktop.cpp

```diff
@@ -76,11 +75,14 @@ void WorkerDesktop::closeWorker(void)
 
     this->close();
 
+    cpuSentinel->ReleaseMinimumFrequency();
```

Comment:
Just to be sure. Is closeWorker() called in every flow of this worker?

Reply:
yes - when we disconnect the USB cable